### ISSUE TITLE
Github bug tracking manager throws 404 exception. 

### ISSUE THREAD

**Issue by nmatra** [ossmeter/ossmeter#49]
I tried to analyse OSSMETER and I got the following exception. 

This exception is thrown during the analysis of the first day.

```
org.eclipse.egit.github.core.client.NoSuchPageException: Not Found (404)
INFO  [ProjectExecutor (ossmeter)] (19:22:00): Last executed: 20130418
    at org.eclipse.egit.github.core.client.PageIterator.next(PageIterator.java:175)
    at org.ossmeter.platform.bugtrackingsystem.github.GitHubManager.getEarliestIssueDate(GitHubManager.java:135)
    at org.ossmeter.platform.bugtrackingsystem.github.GitHubManager.getFirstDate(GitHubManager.java:72)
    at org.ossmeter.platform.bugtrackingsystem.github.GitHubManager.getFirstDate(GitHubManager.java:1)
    at org.ossmeter.platform.delta.bugtrackingsystem.PlatformBugTrackingSystemManager.getFirstDate(PlatformBugTrackingSystemManager.java:72)
    at org.ossmeter.platform.osgi.executors.ProjectExecutor.getLastExecutedDate(ProjectExecutor.java:303)
    at org.ossmeter.platform.osgi.executors.ProjectExecutor.run(ProjectExecutor.java:99)
    at org.ossmeter.platform.osgi.executors.SlaveScheduler$2.run(SlaveScheduler.java:109)
Caused by: org.eclipse.egit.github.core.client.RequestException: Not Found (404)
    at org.eclipse.egit.github.core.client.GitHubClient.createException(GitHubClient.java:552)
    at org.eclipse.egit.github.core.client.GitHubClient.get(GitHubClient.java:740)
    at org.eclipse.egit.github.core.client.PageIterator.next(PageIterator.java:173)
```

To import the project you can use the following:

```
Platform platform = ...
GitHubImporter gi = new GitHubImporter();
gi.importProjectByUrl("https://github.com/ossmeter/ossmeter", platform);
```

**Comment by kolovos** [comment]
Yannis says that this has been fixed in commit 294c0e050c2c952e29e4f49447dbf1fc38138e58. Nikos to test locally.

**Comment by kolovos** [comment]
Duplicate of #7 which is now closed
